kitsune/wiki/views.py

```diff
@@ -1527,7 +1528,10 @@ def get_fallback_locale(doc, request):
     #   * wiki fallbacks for that locale
 
     for locale in all_accepted_locales:
-        if locale in translated_locales:
+        if locale == doc.locale:
```

Comment:
@safwanrahman I think adding en-US to the translations list would cause a different note in the article:
>This page does not exist in German. You have been redirected to the English version instead. If you would like to localize it into German, click here. You can also see the English version of this page.

That would not be very logical. Do you agree?